src/api/app/controllers/webui/packages/assignments_controller.rb

```diff
@@ -1,13 +1,13 @@
 module Webui
   module Packages
     class AssignmentsController < Webui::WebuiController
+      before_action :require_login
```

Comment:
You are not authorizing anything. You just enforce authentication.

Reply:
Check again, please